tiledb/api/examples/using_tiledb_stats.rs

```diff
@@ -0,0 +1,117 @@
+extern crate tiledb;
+use tiledb::{Array, Result as TileDBResult};
+use tiledb::config::Config;
+use tiledb::vfs::VFS;
+use tiledb::Datatype;
+
+
+const ARRAY_NAME: &str = "stats_array";
+const ATTRIBUTE_NAME: &str = "a";
+
+pub fn create_array(row_tile_extent : u32, col_tile_extent : u32) -> TileDBResult<()> {
+    let tdb = tiledb::context::Context::new()?;
+    let config: Config = tiledb::config::Config::new()?;
+    let vfs: VFS = tiledb::vfs::VFS::new(&tdb, &config)?;
+
+    let is_cur_dir = vfs.is_dir(ARRAY_NAME)?;
```

Comment:
The `quickstart_dense` example checks if the array exists already instead of removing the directory every time - to me that seems like a better way to play.  Creating the array isn't really the important part of this example

Reply:
This is a pattern from core because there are lots of issues re-running the test suite so there's a lot of defensive "remove anything to reset state". Maybe we could add a thing that auto deletes arrays using drop traits? Though that's not something I'll ask of this PR.

Reply:
I added `Array::exists`, I think we should use it here.